huaweicloud/services/acceptance/as/resource_huaweicloud_as_instance_acctah_test.go

```diff
@@ -0,0 +1,129 @@
+package as
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+
+	"github.com/chnsz/golangsdk/openstack/autoscaling/v1/instances"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+)
+
+func getASInstanceAttachResourceFunc(cfg *config.Config, state *terraform.ResourceState) (interface{}, error) {
+	region := acceptance.HW_REGION_NAME
+	client, err := cfg.AutoscalingV1Client(region)
+	if err != nil {
+		return nil, fmt.Errorf("error creating AS client: %s", err)
```

Comment:
change AS to autoscaling so that it is consistent?

Reply:
fixed